adapter/internal/oasparser/config_generator.go

```diff
@@ -268,7 +268,7 @@ func castPoliciesToEnforcerPolicies(policies []model.Policy) []*api.Policy {
 			}
 		}
 		enforcerPolicies[i] = &api.Policy{
-			PolicyName: policy.PolicyName,
+			PolicyName: policy.PolicyName, //TODO (renuka): remove this not requited name and version
```

Comment:
Small typo in todo:

Reply:
fixed with https://github.com/wso2/product-microgateway/pull/2754/commits/b7f6b18870e58303c07c2b9ff89afef8a3735571